src/test/java/com/karankumar/bookproject/backend/utils/PredefinedShelfUtilsTest.java

```diff
@@ -40,6 +55,10 @@
     private static Book book3 = new Book("someOtherTitle", NO_AUTHOR, readShelf);
     private static Book book4 = new Book("yetAnotherTitle", NO_AUTHOR, didNotFinishShelf);
 
+    private static final List<String> PREDEFINED_SHELVES = List.of("To read", "Reading", "Read", "Did not finish");
+    private static final List<String> INVALID_SHELVES = List.of("Too read", "Readin", "Do not finish", "Shelf");
+    private static final String ERROR_MSG = "Shelf with name ''{0}'' does not match any predefined shelf";
```

Comment:
Could you please replace 'MSG" with 'MESSAGE'? I know what it means, but others may not